doc/source/whatsnew/v1.0.0.rst

```diff
@@ -853,7 +853,7 @@ Other
 - Fix :class:`AbstractHolidayCalendar` to return correct results for
   years after 2030 (now goes up to 2200) (:issue:`27790`)
 - Bug in :meth:`Series.count` raises if use_inf_as_na is enabled (:issue:`29478`)
-
+- Bug in :meth:`DaataFrame.to_csv` when supplied a series with a ``dtype="string"`` and a ``na_rep``, the ``na_rep`` was being truncated to 2 characters. (:issue:`29975`)
```

Comment:
doesn’t need a separate note as this is not user visible before 1.0, you can instead add this issue number on the same note as where dtype=‘string’ is introduced (above)

Reply:
Will do, thanks!